src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.ReportGenerator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public List<String> generateReport() {
+        Map<String, Fruit> allFruits = Storage.getAllFruits();
+        return allFruits.values().stream()
+                .map(fruit -> fruit.getName() + "," + fruit.getQuantity())
+                .collect(Collectors.toList());
+    }
+}
```

Comment:
According to the checklist, don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition (line 10) and the method definition (line 11).